### DISCUSSION TITLE
Configure chronyd to disable steps?

### DISCUSSION THREAD

**Discussion by velddev** [mysticaltech/terraform-hcloud-kube-hetzner#936] [Ideas]
Heya! we're running a software stack with snowflakes as our main identifiers, and one thing that would be amazign for our nodes to have is to be able to configure chronyd (a ntp provider for linux) to disable steps. 

The steps for this is rather simple, but would be great if this could be automated in an option. 

edit `makestep 1.0 3` to `makestep 0 -1` in `/etc/chronyd.conf`

This ensures that the time steps are disabled, and instead chrony slows down or speeds up the system clock to synchronize to the current most accurate time instead!

**Comment by mysticaltech** [comment]
@velddev Please dive deeper in this use case. I do not see exactly the need for now. If you are open to it and it's generic enought, do not hesitate to shoot a PR please.
